src/main.py

```diff
@@ -0,0 +1,20 @@
+from config import *
+from helpers import *
+import fire
+
+
+# 1. git part
+def get_repo(url, path):
+    """
+    Returns list of commits in repository.
```

Comment:
it definitely does not return anything - it write to file - please fix docstring